build/images/scripts/start_ovs

```diff
@@ -130,9 +130,12 @@ function quit {
     # been restored. The uplink is moved to the OVS bridge to support AntreaFlexibleIPAM mode.
     while [ "`ovsdb-client dump Port|grep antrea-type=uplink`" != "" ]; do
       log_info $CONTAINER_NAME "Uplink found on OVS, wait 1s and retry..."
-      sleep 1 &
-      SLEEP_PID=$!
-      wait $SLEEP_PID
+      # If a second signal (besides SIGKILL) is received, bash will wait for the
+      # sleep to complete, and this function will run again. There is no point
```

Comment:
Out of curiosity: if a second signal is received, when does bash run this function again? after waiting for sleep to complete, then terminate the execution and start over again?

Reply:
That's correct. The second signal will not be handled until the 1-second foreground sleep process exits. At that point the current execution of the `quit` function will stop, and the function will be executed again from the top in response to the second signal.